boot/zephyr/main.c

```diff
@@ -73,7 +81,7 @@ void main(void)
 	SYS_LOG_INF("Setting vector table to %p", vt);
 
 	/* Not all targets set the VTOR, so just set it. */
-	_scs_relocate_vector_table((void *) vt);
+	relocate_vector_table((void *) vt);
```

Comment:
I guess what I'm asking is to just remove the _scs_relocate_vector_table line entirely. It really doesn't belong in the bootloader. Zephyr should be setting the VTOR when it starts, so the applications we run should work fine. If there is a problem there, it should be fixed in Zephyr.